criu/pie/restorer.c

```diff
@@ -1711,6 +1723,10 @@ long __export_restore_task(struct task_restore_args *args)
 	if (ret)
 		goto core_restore_end;
 
+	ret = restore_child_subreaper(&args->child_subreaper);
```

Comment:
Sorry, my bad, I confused you with my previous message... what I meant in my note that the place you called restore_child_subreaper was absolutely right. Should've wrote ``` _after_ CR_STATE_RESTORE_SIGCHLD looks a proper place```